geoh5py/workspace/workspace.py

```diff
@@ -558,35 +457,18 @@ def fetch_children(self, entity: Entity, recursively: bool = False):
                 for kwargs in property_groups.values():
                     recovered_object.find_or_create_property_group(**kwargs)
 
-    def fetch_values(self, uid: uuid.UUID) -> Optional[float]:
+    def fetch_delimiters(
+        self, uid: uuid.UUID
+    ) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
         """
-        Fetch the data values from the source h5file
+        Fetch the delimiter attributes from the source h5file
 
         :param uid: Unique identifier of target data object
 
-        :return value: Array of values
-        """
-        return H5Reader.fetch_values(self.h5file, uid)
-
-    def fetch_vertices(self, uid: uuid.UUID) -> np.ndarray:
-        """
-        Fetch the vertices of an object from the source h5file
-
-        :param uid: Unique identifier of target entity
-
-        :return coordinates: Array of coordinate [x, y, z] locations
-        """
-        return H5Reader.fetch_vertices(self.h5file, uid)
-
-    def fetch_cells(self, uid: uuid.UUID) -> Cell:
-        """
-        Fetch the cells of an object from the source h5file
-
-        :param uid: Unique identifier of target entity
-
-        :return cells: Cell object with vertices index
+        :return (u_delimiters, v_delimiters, z_delimiters):
+            Arrays of delimiters along the u, v, and w axis
```

Comment:
missing period and return format
```suggestion
        :return: Arrays of delimiters along the u, v, and w axis (u_delimiters, v_delimiters, z_delimiters).
```